components/badge/badge.css

```diff
@@ -0,0 +1,23 @@
+.badge {
+  width: 1.5rem;
+  height: 1.5rem;
+  border-radius: 50%;
+  position: absolute;
```

Comment:
suggestion would be to add the badge at bottom right . With some border around.

Reply:
thanks will sure apply